modules/preparing-aws-sts-credentials-for-oadp.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-aws-sts/oadp-aws-sts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="preparing-aws-sts-credentials-for-oadp_{context}"]
+= Preparing AWS STS credentials for OADP
+
+An {aws-full} account must be prepared and configured to accept an {oadp-first} installation. Prepare the {aws-short} credentials by following the proceeding steps.
+
+.Procedure
+. Define the `cluster_name` environment variable by running the following command:
++
+[source,terminal]
+----
+$ export CLUSTER_NAME= <AWS cluster name> <1>
+----
+<1> The variable can be set to any value.
+. Retrieve all the details of the `cluster` such as the `AWS_ACCOUNT_ID, OIDC_ENDPOINT` by running the following command:
++
+[source,terminal]
+----
+$ export CLUSTER_VERSION=$(oc get clusterversion version -o jsonpath='{.status.desired.version}{"\n"}')
+
+export AWS_CLUSTER_ID=$(oc get clusterversion version -o jsonpath='{.spec.clusterID}{"\n"}')
+
+export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o jsonpath='{.spec.serviceAccountIssuer}' | sed 's|^https://||')
+
+export REGION=$(oc get infrastructures cluster -o jsonpath='{.status.platformStatus.aws.region}' --allow-missing-template-keys=false || echo us-east-2)
+
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
```

Comment:
Should there be a + after this, or are 

export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
export ROLE_NAME="${CLUSTER_NAME}-openshift-oadp-aws-cloud-credentials"

supposed to be close together?

Reply:
I think it's supposed to be the way it is currently.

Reply:
OK. Was curious as there is space between all the others except for the last 2, so was just curious. Thanks!